src/FINDOLOGIC/Export/Data/Item.php

```diff
@@ -385,9 +385,6 @@ public function addProperty(Property $property): void
         }
 
         foreach ($property->getAllValues() as $usergroup => $value) {
-            if (!array_key_exists($usergroup, $this->properties)) {
```

Comment:
Why was this removed? Could cause problems.

Reply:
No PHP is smart enough to understand that.

```
php > $a = [];
php > $a['a']['b'] = 'c';
php > var_dump($a);
php shell code:1:
array(1) {
  'a' =>
  array(1) {
    'b' =>
    string(1) "c"
  }
}
```

Also I do not even declare a variable at all and can assign arrays:

```
php > $b['c']['d'] = 'e';
php > var_dump($b);
php shell code:1:
array(1) {
  'c' =>
  array(1) {
    'd' =>
    string(1) "e"
  }
}
```

That's PHP :).